strings/src/main/java/com/revenuecat/purchases/strings/ReceiptStrings.kt

```diff
@@ -0,0 +1,10 @@
+package com.revenuecat.purchases.strings
+
+object ReceiptStrings {
+    const val CHECKING_IF_CACHE_STALE = "Checking if cache is stale AppInBackground %s"
+    const val CLEANING_PREV_SET_HASHED_TOKEN = "[QueryPurchases] Cleaning previously sent tokens"
```

Comment:
Interesting we only do this `[QueryPurchases]` thing here, where we say what the log is about. I think it's such an implementation detail that we can probably remove it

Reply:
Sounds good, I can remove the `[QueryPurchases]` from all the string files